lab

```diff
@@ -26,6 +26,25 @@ E_SCENARIO=3
 #               valid values are: "podified" and "tripleo"
 INSTALLER="podified"
 
+function prepare_inventory_file() {
+    if test -e "/home/student/osp_training/ansible_inventory.yaml"; then
```

Comment:
It's better to negate this condition as you don't do anything if the file exists:
```
if ! test -e; then
   # create the file here
fi

Reply:
Also maybe `-s` would make sense to be used as we want the file to have content at this point